cdap-runtime-ext-dataproc/src/main/java/io/cdap/cdap/runtime/spi/provisioner/dataproc/AbstractDataprocProvisioner.java

```diff
@@ -200,7 +202,7 @@ public Optional<RuntimeJobManager> getRuntimeJobManager(ProvisionerContext conte
       return Optional.empty();
     }
     try {
-      String clusterName = getClusterName(context);
+      String clusterName = getClusterName(context, context.getErrorCategory());
```

Comment:
You should not need to pass category separately

Reply:
removed.